### DISCUSSION TITLE
How to copy directories while keeping structure for formula?

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [Homebrew/discussions#3576] [Everyday usage] [answered]
I have a formula where a binary expects to be able to read from `../<somedir>/lib64/release/somelib.dylib`. I need to install the dylibs from the tarball in the same place, but I am not sure how to do that. `prefix.install` just puts them in the root directory which is not what I want.

**Comment by Rylan12** [comment, marked as answer]
I think this sounds like what you want

```ruby
(prefix/"lib64/release").install "somelib.dylib"
```

**Reply by PythonCoderAS** [reply]
Oh I did not know that any directory (even ones that were derivatives of existing directories) had install methods. Thank you.

**Reply by carlocab** [reply]
`#install` is a `Pathname` instance method, so all `Pathname` objects (e.g. `prefix/"lib64/release"`) have an `install` method.
